nodejs/week1/mealSharing/src/backend/routes/cheap-meals.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+
+const mealsJson = require(__dirname + '/../data/meals.json');
+const reviewsJson = require(__dirname + '/../data/reviews.json');
+
+const cheapMealsArray = mealsJson.filter((cheapMeals) => cheapMeals.price < 50);
+cheapMealsArray.forEach((meal) => {
```

Comment:
And this comment as well:

https://github.com/krithi-kondana/hyf-homework/pull/30#discussion_r382179162

This pattern repeats farther in code, so might be nice to fix all of the places.